### PR TITLE
Make image_pyramids "Support"

### PR DESCRIPTION
**Pull request by mccalluc**
This should not be merged before https://github.com/hubmapconsortium/portal-ui/pull/1690 is in place. The umbrella issue is https://github.com/hubmapconsortium/portal-ui/issues/1576. Filing as draft for now: @ilan-gold , if you have thoughts?

### CODE REVIEW

File: src/elasticsearch/addl_index_transformations/portal/reset_entity_type.py
Comment (ilan-gold) [inline src/elasticsearch/addl_index_transformations/portal/reset_entity_type.py:14]: Is the idea we would not display `Support` types?

Comment (mccalluc) [reply src/elasticsearch/addl_index_transformations/portal/reset_entity_type.py:14]: yes... old urls would still work, but we don't want to show them in the list of search results.

**Comment by mccalluc** [comment]
@john-conroy -- I'm going to start looping you in on indexing changes. This PR is tiny, but please let me know if you have questions about the bigger picture.

**Review by john-conroy — APPROVED** [review]

File: src/elasticsearch/addl_index_transformations/portal/reset_entity_type.py
Comment (mccalluc) [reply src/elasticsearch/addl_index_transformations/portal/reset_entity_type.py:20]: It's clear that we are downstream from entity-api, but can you clarify what you mean when you say "search-api would be able to understand this entity type"? Does the entity type matter as search-api is proxying requests to Elasticsearch?

Comment (yuanzhou) [reply src/elasticsearch/addl_index_transformations/portal/reset_entity_type.py:20]: The entity-api has no knowledge about this `Support` entity_type. When the search-api's indexer handles the index/reindex by making calls to entity-api, it'll only get back the defined entity types: Donor, Sample, Dataset, and Collection. If you change the `entity_type` from "Dataset" to "Support" when transforming the entity during indexing, the count of Dataset docs indexed on `/dev-search` page will probably be less then the actual number.

And when the portal-ui makes a call to entity-api to fetch the provenance of a entity with "Support" type, the entity-api will throw validation errors. 

We want to make sure these won't cause issues.

Comment (mccalluc) [reply src/elasticsearch/addl_index_transformations/portal/reset_entity_type.py:20]: > When the search-api's indexer handles the index/reindex by making calls to entity-api, it'll only get back the defined entity types: Donor, Sample, Dataset, and Collection. 

Right: This is a change the entity-api does not need to worry about: It's a transformation of the data that comes from entity-api.

> the count of Dataset docs indexed on /dev-search page will probably be less then the actual number

Right, that is desired: We don't want these "Support" items to show up in searches for Datasets: They are only there to provide information for visualization. We want to make sure they are still in the index, so old links still work, but there would be no easy way to find or navigate to them in future, and that's by design.

Thanks for asking questions. The linked issues try to provide a bigger picture about what the goal of "vis-lifting" is, or if there are things that still don't seem right, please let me know.

**Review by yuanzhou — APPROVED** [review]

**Comment by mccalluc** [comment]
@yuanzhou -- Ping me when the results of a reindex are on TEST, or let me know if you have other concerns? No rush. Thanks!

**Comment by yuanzhou** [comment]
@mccalluc I was gonna merge and redeploy on TEST with reindex, but we are still waiting for @ilan-gold 's approval.

**Comment by ilan-gold** [comment]
@yuanzhou Our PR custom is that a "comment" is as good as an approval.

**Comment by yuanzhou** [comment]
@mccalluc the changes have been deployed on TEST and reindex went well

**Comment by mccalluc** [comment]
@yuanzhou -- Great! We need to do a little bit of UI work to catch up to this. I'll post an update here and on slack.

**Pull request merged**
